tests/integration/test_api.py

```diff
@@ -569,3 +569,40 @@ def test_prediction_collection_no_filter(client):
     assert prediction_data[1]["id"] == prediction2.id
     assert prediction_data[1]["type"] == "brand"
     assert prediction_data[1]["value_tag"] == "en:beers"
+
+
+def test_get_unanswered_questions_api(client):
+    result = client.simulate_get("/api/v1/questions/unanswered/")
+
+    assert result.status_code == 200
+    assert result.json == {"questions": [], "status": "no_questions"}
+
+    product1 = ProductInsightFactory(type="category", annotation=0)
+    result = client.simulate_get("/api/v1/questions/unanswered/", 
+        params={
+            "count": 5,
+            "page": 1,
+            "question_type": "category",
```

Comment:
Hey @alexgarel 

I am not able to pass the parameters to the functions. `question_type` is always `None` when passed to `get_insights`

Reply:
It's working, but I think you pdb in the first `simulate_get` above at line 576, where this parameter is None ! (that's why this is a good idea to separate tests ;-) )